K2Bridge.Tests.UnitTests/Visitors/VisitorTestsUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Tests.UnitTests.Visitors
+{
+    using global::UnitTests.K2Bridge.Visitors;
+    using K2Bridge.Models.Request;
+    using K2Bridge.Models.Request.Queries;
+    using K2Bridge.Visitors;
+
+    public static class VisitorTestsUtils
+    {
+        /// <summary>
+        /// A helpler method to add the default DSL to visitor in order to
+        /// Have the ISchemaRetriever initialised by the default visitor.
+        /// </summary>
+        /// <param name="visitor"></param>
+        internal static void AddRootDsl(ElasticSearchDSLVisitor visitor)
+        {
+            var dsl = new ElasticSearchDSL
+            {
+                Query = new Query
+                {
+                    Bool = new BoolQuery(),
+                },
+                IndexName = "someindex",
+            };
+            visitor.Visit(dsl);
```

Comment:
also, the method says: AddRootDsl
but at the end you call visit... so its much more that just add root. it actually visits.

Reply:
also, I think you shouldnt use visitor.Visit but dsl.Accept(visitor)

Reply:
I renamed the method, however, I kept Visit since I am not changing existing tests code.